ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/91521_SCAPP_apply-to-become/98085_integration-tests.js

```diff
@@ -0,0 +1,1972 @@
+/// <reference types ='Cypress'/>
+
+describe('Fetch data from Internal', () => {
+    let fetchProjects = ['2050']
+    let url = Cypress.env('url')
+
+    beforeEach(() => {
+		cy.login()
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflict issues weren't fully cleaned up

Reply:
Yeah, I noticed that on rebasing - sorting that out now.

Reply:
I've reverted that commit so shouldn't see any conflict's now.